specs-go/version.go

```diff
@@ -11,7 +11,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = "-rc6-dev"
+	VersionDev = "-dev"
```

Comment:
Up until now, `-dev` has meant “working towards”, so `1.0.0-rc6-dev` was followed by `1.0.0-rc6`.  I'm not sure how we want to handle the post-1.0 world where it's not clear what release we're working on (`2.0.0-rc1`?  `1.1.0-rc1`?  `1.0.1-rc1`?)  I'd guess master will most likely target `1.1.0-rc1`, so maybe you want to setup `1.1.0-rc1-dev` in the tip commit of this PR?